docs/apache-airflow-providers-cncf-kubernetes/operators.rst

```diff
@@ -15,114 +15,1015 @@
+      ----    ------     ----  ----               -------
+      Normal  Scheduled  15m   default-scheduler  Successfully assigned default/airflow-pod-operator
+                                                  -aed97ecd64854367ad7d0ff39f37859c to minikube
+      Normal  Pulling    15m   kubelet            Pulling image "alpine"
+      Normal  Pulled     15m   kubelet            Successfully pulled image "alpine" in 4.214686688s
+      Normal  Created    15m   kubelet            Created container base
+      Normal  Started    15m   kubelet            Started container base
 
-.. note::
-  The :doc:`Kubernetes executor <apache-airflow:executor/kubernetes>` is **not** required to use this operator.
 
-How does this operator work?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-The :class:`~airflow.providers.cncf.kubernetes.operators.kubernetes_pod.KubernetesPodOperator` uses the
-Kubernetes API to launch a pod in a Kubernetes cluster. By supplying an
-image URL and a command with optional arguments, the operator uses the Kube Python Client to generate a Kubernetes API
-request that dynamically launches those individual pods.
-Users can specify a kubeconfig file using the ``config_file`` parameter, otherwise the operator will default
-to ``~/.kube/config``.
 
-The :class:`~airflow.providers.cncf.kubernetes.operators.kubernetes_pod.KubernetesPodOperator` enables task-level
-resource configuration and is optimal for custom Python
-dependencies that are not available through the public PyPI repository. It also allows users to supply a template
-YAML file using the ``pod_template_file`` parameter.
-Ultimately, it allows Airflow to act a job orchestrator - no matter the language those jobs are written in.
+Defining Environment Variables for Pod
+======================================
 
-How to use cluster ConfigMaps, Secrets, and Volumes with Pod?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To add ConfigMaps, Volumes, and other Kubernetes native objects, we recommend that you import the Kubernetes model API
-like this:
+- Creating Task using KubernetesPodOperator with given environment variables.
 
 .. code-block:: python
 
-  from kubernetes.client import models as k8s
-
-With this API object, you can have access to all Kubernetes API objects in the form of python classes.
-Using this method will ensure correctness
-and type safety. While we have removed almost all Kubernetes convenience classes, we have kept the
-:class:`~airflow.kubernetes.secret.Secret` class to simplify the process of generating secret volumes/env variables.
-
-.. exampleinclude:: /../../airflow/providers/cncf/kubernetes/example_dags/example_kubernetes.py
-    :language: python
-    :start-after: [START howto_operator_k8s_cluster_resources]
-    :end-before: [END howto_operator_k8s_cluster_resources]
-
-Difference between ``KubernetesPodOperator`` and Kubernetes object spec
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Why are you getting rid of this?

Reply:
I will added text description from old doc.